dashboard/test/helpers/levels_helper_test.rb

```diff
@@ -33,44 +32,44 @@ def request
   end
 
   test "non-custom level displays localized instruction after locale switch" do
-    DEFAULT_LOCALE = 'en-us'
```

Comment:
uppercase is constant, lowercase is normal variable. I was explaining why I made all those constants normal variables.